gap/PackageManager.gi

```diff
@@ -1057,7 +1057,11 @@ function()
   if PKGMAN_CustomPackageDir <> "" then
     dir := PKGMAN_CustomPackageDir;
   else
-    dir := UserHomeExpand("~/.gap/pkg");  # TODO: cygwin?
+    if GAPInfo.UserGapRoot = fail then
+      UserHomeExpand("~/.gap/pkg"); # Fallback
```

Comment:
That's wrong... anyway your explanation with the error also makes sense, so feel free to return to that